COLLABORATOR_GUIDE.md

```diff
@@ -519,9 +501,26 @@ Save the file and close the editor. You'll be asked to enter a new
 commit message for that commit. This is a good moment to fix incorrect
 commit logs, ensure that they are properly formatted, and add
 `Reviewed-By` lines.
+
 * The commit message text must conform to the
 [commit message guidelines](./CONTRIBUTING.md#commit-message-guidelines).
 
+* Always modify the original commit message to include additional meta
+information regarding the change process:
+
+  * Required: A `PR-URL:` line that references the *full* GitHub URL of the
+    original pull request being merged so it's easy to trace a commit back to
+    the conversation that led up to that change.
+  * Required: A `Reviewed-By: Name <email>` line for yourself and any
```

Comment:
I thought the order in the previous revision was invariant, i.e.:
1. PR-URL
2. [Fixes]
3. [Refs]
4. Reviewed-By

Anyway that's the order `node-review` formats the metadata.

Reply:
@refack I didn't mean for the list to specify ordering, but since that's something you inferred and I doubt you will be alone in doing that, I've change it back to the ordering you have above.